client/src/components/game/Game.tsx

```diff
@@ -191,20 +190,13 @@ function Game() {
           </>
         )}
         {inDoneGuessing() && (
-          <Answer
-            correctChoice={(state.answer as IAnswer).correctChoice}
+          <RoundResults correctChoice={(state.answer as IAnswer).correctChoice}
             players={(state.answer as IAnswer).players}
-            outOfChunks={false}
-            repository={(state.answer as IAnswer).repository}
-          />
+            repository={(state.answer as IAnswer).repository} />
         )}
         {inOutOfChunks() && (
-          <Answer
-            correctChoice={(state.answer as IAnswer).correctChoice}
-            players={SessionService.getSortedPlayers(state.players)}
-            outOfChunks={true}
-            repository={(state.answer as IAnswer).repository}
-          />
+          <GameResults players={(state.answer as IAnswer).players}
+            endGameMessage="Out of chunks for you to guess on. Thanks for playing!" />
```

Comment:
I added it for now since I don't know if we should display a message once the game ends. The players may not be aware that the game ended because they ran out of chunks